packages/acs-ui-common/src/safeStringify.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Wrap JSON.stringify in a try-catch as JSON.stringify throws an exception if it fails.
+ *
+ * Use this only in areas where the JSON.stringify is non-critical and OK for the JSON.stringify to fail, such as logging.
+ *
+ * @internal
+ */
+export const _safeStringify = (
```

Comment:
might I suggest changing it to `_safeJSONStringify` just so that there is absolute clarity about what it is used for just by reading the name.

Reply:
Yep I like that better, updated